time-tracker-webapi/src/TimeTracker.Api/Views/AdminReports/AllUsersReport.cshtml

```diff
@@ -0,0 +1,37 @@
+@model System.Collections.Immutable.IImmutableList<TimeTracker.Library.Models.Admin.UserReport>
+
+@{
+    ViewBag.Title = "Users YTD Hour Report";
+    Layout = "_Layout";
+}
+
+<h2>Users YTD Hour Report</h2>
+
+<table>
+    <thead>
+    <tr>
+        <th>Username</th>
```

Comment:
Yes there is! I've been going back and forth on this, but at the end I ended up settling on wanting the native HTML to drive the UI as much as possible and avoid server side magic rendered things when it makes sense. For example, at some point this view could be replaced with a javascript driven view that just calls and API, and then it would bee fully html driven.